airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -88,7 +79,7 @@ def get_task_instance(
         )
     if task_instance is None:
         raise NotFound("Task instance not found")
-    if task_instance[0].map_index != -1:
+    if task_instance.map_index != -1:
```

Comment:
Oops!

Reply:
No this is intentional, it's a consequence of the queries no longer adding entities it's no longer a tuple

Reply:
I meant to oops the previous implementation; it should not have been written that way in the first place 🙂

Reply:
Well I think it _was_ functional 🤷. Albeit confusing!